code/Row.py

```diff
@@ -0,0 +1,7 @@
+class Row:
```

Comment:
Remove this class. Lets add all these small utilities in a single Utils.py file.

Reply:
Since its a separate class, let it be in a new file.